page/src/scss/general.scss

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&display=swap');
```

Comment:
Either add a cookie policy or add fonts directly in the project as not compliant with GDPR.